templates/partials/_video.html

```diff
@@ -31,5 +28,25 @@
 
       asciicastplayerFrame.appendChild(script);
     }
+
+    if (document.querySelector("[data-youtube]")) {
+      const videoFrame = document.createElement("iframe");
+      videoFrame.id = "ytplayer";
+      videoFrame.type = "text/html";
+      videoFrame.width = "818";
+      videoFrame.height = "460";
+      videoFrame.src = "{{ video.url }}?autoplay=0&cc_load_policy=1&modestbranding=1&rel=0";
+
+      const videoWrapper = document.getElementById("youtube-video-wrapper");
```

Comment:
You are mixing your lookups by checking for [data-youtube] then selecting the element with the `id`. You would set one as a variable above and use everywhere you need the wrapper. Protects from someone changing one and breaking this and saves traversing the dom twice.

Reply:
Of course. What was I thinking? Fixed.